DLA-Future (#3)
RMeli/cp2k: CMakeLists.txt

option(CP2K_USE_SPLA
       "Use SPLA offloading gemm feature to the GPU if it is beneficial. " OFF)
option(CP2K_USE_METIS "enable metis library support" OFF)

> option(CP2K_USE_DLAF "Use DLA-Future Cholesky decomposition and eigensolver"

Reviewer: ```suggestion
option(CP2K_USE_DLAF "Use DLA-Future for Cholesky decomposition and eigensolver"
```
?

Author: Mmmh... I agree that how I wrote it might sound a bit off. But with "for eigensolver" doesn't sound right either. Maybe
```suggestion
option(CP2K_USE_DLAF "Use DLA-Future for Cholesky decompositions and the solution of eigenvalue problems"
```

Reviewer: Yeah, good point! I like that formulation as well. I was considering commenting with
```suggestion
option(CP2K_USE_DLAF "Enable DLA-Future support"
```
since that's what most of the other options say, but it probably doesn't hurt to be a bit more explicit here about what dlaf is used for!

Author: I went for the first option. We can maybe switch to the second one if we will use DLA-Future GEMM too.

  find_package(LibVORI REQUIRED)
endif()

if(CP2K_USE_DLAF)
  find_package(DLAF REQUIRED)

> endif()

Reviewer: I think `cp2kConfig.cmake` will need a corresponding change.

Author: Good catch, thanks!
